modules/test/playwright/tests/analytics-web/cookiesPreferenceHandling.spec.ts

```diff
@@ -0,0 +1,192 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginAnalyticsCloudTest} from '../../fixtures/loginAnalyticsCloudTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {liferayConfig} from '../../liferay.config';
+import {HomePage} from '../../pages/portal-web/HomePage';
+import {
+	connectToAnalyticsCloud,
+	disconnectFromAnalyticsCloud,
+	goToAnalyticsCloudInstanceSettings,
+	syncAllContacts,
+	syncSite,
+} from '../analytics-settings-web/utils/analyticsSettings';
+import {createDataSource} from '../osb-faro-web/utils/dataSource';
+
+export const test = mergeTests(
+	loginAnalyticsCloudTest(),
+	loginTest(),
+	featureFlagsTest({
+		'LPD-10588': true,
+	})
+);
+
+async function changeCookiePreferenceHandling(
+	page,
+	{enableCookie, enableExplicitCookieConsentMode}
+) {
+	const homepage = new HomePage(page);
+
+	await homepage.openApplicationMenu();
+
+	await page.getByRole('tab', {name: 'Control Panel'}).click();
+
+	await page.getByRole('menuitem', {name: 'Instance Settings'}).click();
+
+	await page.getByRole('link', {name: 'Cookies'}).click();
+
+	if (enableCookie) {
+		await page.getByLabel('Enabled').check();
+	} else {
+		await page.getByLabel('Enabled').uncheck();
+	}
+
+	if (enableExplicitCookieConsentMode) {
+		await page.getByLabel('Explicit Cookie Consent Mode').check();
+	} else {
+		await page.getByLabel('Explicit Cookie Consent Mode').uncheck();
+	}
+
+	await page.getByRole('button', {name: 'Update'}).click();
+
+	await page
+		.getByText('Success:Your request completed successfully.')
+		.click();
+}
```

Comment:
Hey @interaminense, The cookie page is not within the scope of the AC, so I think this one should not be part of the AC folders. I think this function and the file should be inside some folder of the DXP team that is responsible for this.

I think the tests you created make sense to be in the AC folder, but I think the file should have another name related to events, because that's what you're checking here. What do you think?

Reply:
Hey @interaminense, Why don't you use the `analyticsSettingsWebConfig` folder instead of creating another folder? `analyticsSettingsWebConfig` folder should represent the tests that are being executed considering that the DXP is the most important part, so would this be the same case as yours here or not?

Reply:
Changes were done in the analytics-web dist instead analytics-settings-web, I agree but I'm following Playbook pattern